templates/helpers/jsesc.js

```diff
@@ -0,0 +1,5 @@
+var jsesc = require('jsesc');
+
+module.exports = (value) => jsesc(
+    value.replace(/[\r\n]{1,2}/g, '')
```

Comment:
My original idea was to use `jsesc(value, { wrap: true })` so it returns a string literal including the quotes.